### DISCUSSION TITLE
SPECIFIC environment variable is undefined -- dozens other are working

### DISCUSSION THREAD

**Discussion by OmerWow** [vercel/community#685] [Help] [answered]
Hello everyone,

So I'm developing a project for years now, and i'm using dozens of env vars for many specific branches, some are server only, some NEXT_PUBLIC.
All of them are working in all branches and all cases.

I do however have a single recently added env var that is undefined on the branch I assigned it to and I have no idea why.

I redeployed many times after adding this var, and neither Next.js 1.12.5 nor 1.12.6 (latest) fixed the issue.

This really looks like a Vercel problem, because it started when AWS and Vercel went about a week ago.
It then worked for about a day, and now it's undefined ever since.

This is really weird as it is the only var out of many that is not working in production.

On localhost, EXACT same code base but with env vars that are being pulled from .env.local it is working fine as well.

**Comment by swarnava** [comment, marked as answer]
Are you able to load read those environment variable in local environment? if yes, Can you create a support case using https://vercel.com/support/request ?
